launchdarkly-server-sdk/src/data_source.rs

```diff
@@ -196,6 +200,9 @@ impl DataSource for StreamingDataSource {
                         if let Err(e) = stored {
                             init_success = false;
                             error!("error processing update: {:?}", e);
+                        } else {
+                            // Report event as stored at the current time.
+                            event_processed_at(std::time::SystemTime::now())
```

Comment:
This doesn't seem like the right spot to call the callback! At least, it will only call the callback for actual events, and will _not_ call the callback for comments. So we're not going to be notified of keepalive events.

Reply:
Urgh, I didn't read the code carefully enough to notice that the comments are ignored :(